Python/opcode_cache.h

```diff
@@ -0,0 +1,137 @@
+#ifndef Py_OPCODE_CACHE_H
+#define Py_OPCODE_CACHE_H
+
+#ifdef Py_DEBUG
+// --with-pydebug is used to find memory leak.  opcache makes it harder.
+// So we disable opcache when Py_DEBUG is defined.
+// See bpo-37146
+#define OPCACHE_MIN_RUNS 1024  /* disable opcache */
+#else
+#define OPCACHE_MIN_RUNS 1024  /* create opcache when code executed this time */
+#endif
+#define OPCODE_CACHE_MAX_TRIES 20
+#define OPCACHE_STATS 1  /* Enable stats */
+
+#if OPCACHE_STATS
+extern size_t opcache_code_objects;
+extern size_t opcache_code_objects_extra_mem;
+
+extern size_t opcache_global_opts;
+extern size_t opcache_global_hits;
+extern size_t opcache_global_misses;
+
+extern size_t opcache_attr_opts;
+extern size_t opcache_attr_hits;
+extern size_t opcache_attr_misses;
+extern size_t opcache_attr_deopts;
+extern size_t opcache_attr_total;
+#endif
+
+/* macros for opcode cache */
+#define OPCACHE_CHECK() \
```

Comment:
These no longer have to be a macro. They can be inlinable static funcs.

Reply:
:+1: Good idea, will convert them!